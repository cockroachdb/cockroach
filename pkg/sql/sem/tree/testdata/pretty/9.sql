SELECT ((((((1, '2', 3) AS a, b, c), ((4,'5') AS a, b), (ROW(6) AS a)) AS a, b, c), ((7, 8) AS a, b), (ROW('9') AS a)) AS a, b, c) AS r
