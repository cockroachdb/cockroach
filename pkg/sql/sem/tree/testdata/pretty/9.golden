1:
-
SELECT
	(
		(
			(
				(
					(
						(
							1,
							'2',
							3
						)
						AS
							a,
							b,
							c
					),
					(
						(
							4,
							'5'
						)
						AS
							a,
							b
					),
					(
						ROW(
							6
						)
						AS
							a
					)
				)
				AS
					a,
					b,
					c
			),
			(
				(
					7,
					8
				)
				AS
					a,
					b
			),
			(
				ROW(
					'9'
				)
				AS
					a
			)
		)
		AS
			a,
			b,
			c
	)
		AS r

8:
--------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS
							   a
					   )
				   )
				   AS
					   a,
					   b,
					   c
			   ),
			   (
				   (
					   7,
					   8
				   )
				   AS
					   a,
					   b
			   ),
			   (
				   ROW(
					   '9'
				   )
				   AS
					   a
			   )
		   )
		   AS
			   a,
			   b,
			   c
	   )
		   AS r

21:
---------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS
							   a
					   )
				   )
				   AS
					   a,
					   b,
					   c
			   ),
			   (
				   (
					   7,
					   8
				   )
				   AS
					   a,
					   b
			   ),
			   (
				   ROW(
					   '9'
				   )
				   AS
					   a
			   )
		   )
		   AS a, b, c
	   )
		   AS r

23:
-----------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS
							   a
					   )
				   )
				   AS
					   a,
					   b,
					   c
			   ),
			   (
				   (
					   7,
					   8
				   )
				   AS
					   a,
					   b
			   ),
			   (
				   ROW(
					   '9'
				   )
				   AS a
			   )
		   )
		   AS a, b, c
	   )
		   AS r

25:
-------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS
							   a
					   )
				   )
				   AS
					   a,
					   b,
					   c
			   ),
			   (
				   (7, 8)
				   AS
					   a,
					   b
			   ),
			   (
				   ROW(
					   '9'
				   )
				   AS a
			   )
		   )
		   AS a, b, c
	   )
		   AS r

26:
--------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS
							   a
					   )
				   )
				   AS
					   a,
					   b,
					   c
			   ),
			   (
				   (7, 8)
				   AS a, b
			   ),
			   (
				   ROW(
					   '9'
				   )
				   AS a
			   )
		   )
		   AS a, b, c
	   )
		   AS r

27:
---------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS
							   a
					   )
				   )
				   AS
					   a,
					   b,
					   c
			   ),
			   (
				   (7, 8)
				   AS a, b
			   ),
			   (
				   ROW('9')
				   AS a
			   )
		   )
		   AS a, b, c
	   )
		   AS r

29:
-----------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS
							   a
					   )
				   )
				   AS a, b, c
			   ),
			   (
				   (7, 8)
				   AS a, b
			   ),
			   (
				   ROW('9')
				   AS a
			   )
		   )
		   AS a, b, c
	   )
		   AS r

30:
------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS
							   a
					   )
				   )
				   AS a, b, c
			   ),
			   (
				   (7, 8)
				   AS a, b
			   ),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

31:
-------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS a
					   )
				   )
				   AS a, b, c
			   ),
			   (
				   (7, 8)
				   AS a, b
			   ),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

32:
--------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(
							   6
						   )
						   AS a
					   )
				   )
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

33:
---------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS
							   a,
							   b
					   ),
					   (
						   ROW(6)
						   AS a
					   )
				   )
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

34:
----------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (
							   4,
							   '5'
						   )
						   AS a, b
					   ),
					   (
						   ROW(6)
						   AS a
					   )
				   )
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

35:
-----------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (4, '5')
						   AS a, b
					   ),
					   (
						   ROW(6)
						   AS a
					   )
				   )
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

36:
------------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS
							   a,
							   b,
							   c
					   ),
					   (
						   (4, '5')
						   AS a, b
					   ),
					   (ROW(6) AS a)
				   )
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

37:
-------------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (
							   1,
							   '2',
							   3
						   )
						   AS a, b, c
					   ),
					   (
						   (4, '5')
						   AS a, b
					   ),
					   (ROW(6) AS a)
				   )
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

38:
--------------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (1, '2', 3)
						   AS a, b, c
					   ),
					   (
						   (4, '5')
						   AS a, b
					   ),
					   (ROW(6) AS a)
				   )
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

42:
------------------------------------------
SELECT (
		   (
			   (
				   (
					   (
						   (1, '2', 3)
						   AS a, b, c
					   ),
					   ((4, '5') AS a, b),
					   (ROW(6) AS a)
				   )
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

48:
------------------------------------------------
SELECT (
		   (
			   (
				   (
					   ((1, '2', 3) AS a, b, c),
					   ((4, '5') AS a, b),
					   (ROW(6) AS a)
				   )
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

80:
--------------------------------------------------------------------------------
SELECT (
		   (
			   (
				   (((1, '2', 3) AS a, b, c), ((4, '5') AS a, b), (ROW(6) AS a))
				   AS a, b, c
			   ),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

90:
------------------------------------------------------------------------------------------
SELECT (
		   (
			   ((((1, '2', 3) AS a, b, c), ((4, '5') AS a, b), (ROW(6) AS a)) AS a, b, c),
			   ((7, 8) AS a, b),
			   (ROW('9') AS a)
		   )
		   AS a, b, c
	   )
		   AS r

122:
--------------------------------------------------------------------------------------------------------------------------
SELECT (
		   (((((1, '2', 3) AS a, b, c), ((4, '5') AS a, b), (ROW(6) AS a)) AS a, b, c), ((7, 8) AS a, b), (ROW('9') AS a))
		   AS a, b, c
	   )
		   AS r

131:
-----------------------------------------------------------------------------------------------------------------------------------
SELECT ((((((1, '2', 3) AS a, b, c), ((4, '5') AS a, b), (ROW(6) AS a)) AS a, b, c), ((7, 8) AS a, b), (ROW('9') AS a)) AS a, b, c)
		   AS r

136:
----------------------------------------------------------------------------------------------------------------------------------------
SELECT ((((((1, '2', 3) AS a, b, c), ((4, '5') AS a, b), (ROW(6) AS a)) AS a, b, c), ((7, 8) AS a, b), (ROW('9') AS a)) AS a, b, c) AS r
